x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/esql_query_expression.tsx

```diff
@@ -194,11 +194,8 @@ export const EsqlQueryExpression: React.FC<
             setParam('esqlQuery', q);
             refreshTimeFields(q);
           }}
-          expandCodeEditor={() => true}
-          isCodeEditorExpanded={true}
           onTextLangQuerySubmit={async () => {}}
           detectedTimestamp={detectedTimestamp}
-          hideMinimizeButton={true}
```

Comment:
Could we also restore the borders in the stack manager rule form?

<img width="602" alt="Screenshot 2024-07-31 at 10 46 47 AM" src="https://github.com/user-attachments/assets/16d5da7a-f643-4cf5-83e9-5945bad31682">

Reply:
Sure done 

<img width="646" alt="image" src="https://github.com/user-attachments/assets/13b65aeb-5df2-47a7-b01c-66ff29a434d6">